lib/src/widget_stage.dart

```diff
@@ -66,6 +101,46 @@ class _WidgetStageState extends State<WidgetStage> {
   }
 }
 
+class _LeadingFieldConfigurationWidget extends StatelessWidget {
+  const _LeadingFieldConfigurationWidget({
+    required this.title,
+    required this.configurator,
+  });
+
+  final FieldConfigurator configurator;
+  final String title;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Container(
+          padding: const EdgeInsets.only(bottom: 4.0),
+          decoration: BoxDecoration(
+            border: Border(
+              bottom: BorderSide(color: Colors.grey.shade400),
+            ),
+          ),
+          width: double.infinity,
```

Comment:
The infinite width can be removed

Reply:
No, that would result in a line that is as wide as the title instead of being as wide as the configuration bar.